rpms/generate_rpm_package.sh

```diff
@@ -241,6 +242,10 @@ main() {
                 help 1
             fi
             ;;
+        "-src"|"--source")
```

Comment:
This option should be `--src` only.

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/322/commits/1c0fc432f8a6a01bc925799249c0761244cb4f1f